src/main/java/DatabaseInteractin/Client.java

```diff
@@ -0,0 +1,99 @@
+package DatabaseInteractin;
+
+import generated.addProductRequest;
+import generated.addProductResponse;
+import generated.addShoppingCartRequest;
+import generated.addShoppingCartResponse;
+import generated.listByIdRequest;
+import generated.listByIdResponse;
+import generated.listProductRequest;
+import generated.listProductResponse;
+import generated.productGrpc;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+
+import java.sql.SQLException;
+import java.util.Iterator;
+import java.util.Scanner;
+
+public class Client {
+  public static void main(String[] args) throws SQLException {
+    ManagedChannel channel = ManagedChannelBuilder.forAddress("localhost", 50051).usePlaintext().build();
+    productGrpc.productBlockingStub prodStub = productGrpc.newBlockingStub(channel);
```

Comment:
`productGrpc` and other generated classes look like variables because of wrong casing... classes should be Capitalized